keps/sig-node/4603-tune-crashloopbackoff/README.md

```diff
@@ -224,12 +220,8 @@ are considered too conservative, especially in cases where the exit code was 0
 length of the pod run is less than 10 minutes.
 
 This KEP proposes the following changes:
-* Provide 2 alpha-gated changes to get feedback and periodic scalability tests
-  * Changes to the global initial backoff to 1s and maximum backoff to 1 minute
-  * A knob to cluster operators to configure maximum backoff down, to minimum 1s, at the node level
-* Formally split image pull backoff and container restart backoff behavior
-* Formally split backoff counter reset threshold for container restart backoff
-  behavior and maintain the current 10 minute recovery threshold
+* Provide an alpha-gated change to get feedback and periodic scalability tests
+  on changes to the global initial backoff to 1s and maximum backoff to 1 minute
```

Comment:
Consider adding a sentence to both Overviews about how this was originally a bigger KEP that has been split out into two, and link to the other one there, so its quickly in context for new or returning readers.

Reply:
Done.